electives/18349.md

```diff
@@ -45,9 +49,9 @@ Some major topics that are tested on exams are
 
 ## How to do well
 
-- Figure out a way to debug your code on your Raspberry Pi early on.
+- Figure out a way to debug your code on your STM32 board early on. 
+- Having a good project partner is very important for this course, especially for those who are inexperienced with embedded programming. The workload will be very overwhelming otherwise
 - Datasheets suck, but you should dedicate yourself to learning how to read what you need to know. They are also almost intentionally ambiguous in certain parts, so sometimes you will just have to try things out.
 - Sometimes exams have trivia, so you should write down some trivia on your cheatsheet. There is quite a heavy weight on multiple choice and short answer questions, so trivia can be worth a lot. Trivia includes things like history, how certain devices work or applications for them, definitions of particular concepts.
-- Learn how to use `tmux`, at least to the point where you can have multiple tabs and windows, and save sessions
-- Learn to use `vim`. You'll be much more efficient when programming on your Raspberry Pi. If you program on your own computer, you'll constantly have to upload code to the Pi, which can be slow. If you work on the Pi directly, there's no need to transfer code from your laptop to the Pi.
```

Comment:
Is it not the case that you guys use terminal to develop anymore? If not, what do you use now?